src/SonarScanner.MSBuild.PreProcessor/WebClientDownloader.cs

```diff
@@ -47,7 +48,19 @@ public WebClientDownloader(string userName, string password, ILogger logger)
 
             if (this.client == null)
             {
-                this.client = new HttpClient();
+                if (clientCertPath != null)
+                {
+                    var cert = new X509Certificate2(clientCertPath);
+
+                    var _clientHandler = new HttpClientHandler();
```

Comment:
i would have put clientHandler instead, withouth the underscore, just to keep consistency in the naming.

Reply:
Done, I sent today